simple_api/object/validators.py

```diff
@@ -0,0 +1,73 @@
+from inspect import isclass
+
+
+def build_validation_fn(action_validators, field_validators):
+    """
+    Prepares function for validation of input values and the action overall, instantiates validators which are not
+    already instantiated. Action_validators can be a sigle validator, Field_validators a List of tuples
+    (string, (validators...)) as prepared in object/actions.py
+    """
+    if action_validators and field_validators is None:
+        return None
```

Comment:
jsi si jisty, ze tohle dela to, co chces?

Reply:
Přehlídnutí se 😨